spec/config/initializers/hyrax_actor_factory_spec.rb

```diff
@@ -9,28 +9,29 @@
     subject(:middlewares) { actor_factory.middlewares }
 
     it do
-      expect(middlewares.count).to eq 20
+      expect(middlewares.count).to eq 21
```

Comment:
This spec is fine, but it needs to be moved since the file config/initializers/hyrax_actor_factory.rb has been removed and the code put into config/application. Not a big deal, but should be done at some point.